Fixed integrity left/right aux

DatedGore opened the pull request:
Fixed a issue where it would skip sprites.

DatedGore committed:
Fixed integrity left/right aux

Fixed a issue where it would skip sprites.

creator2013 reviewed (changes requested):
- Remove the empty line at the top
- Fix the tab spaces in COMPONENT.Patterns

DatedGore committed:
Fixed Fed Sig Integrity left/right aux

Fixed the left/right aux to where it wouldnt skip sprites.

DatedGore committed:
Fixed left/right Fed Sig Integrity

Fixed the left/right aux mode for Federal Signal Integrity so it wouldnt skip sprites

DatedGore committed:
cruise comment go bye bye

creator2013 reviewed (approved):
Looks good to me

DatedGore commented:
ok.
